src/desktop/space/mod.rs

```diff
@@ -121,12 +123,37 @@ impl Space {
         self.windows.iter()
     }
 
+    /// Finds the topmost surface under this point if any and returns it
+    /// together with the location of this surface and its window.
+    pub fn surface_under<P: Into<Point<f64, Logical>>>(
```

Comment:
Let's write in the documentation that this takes input regions into account to find the surface.

Reply:
Yeah, I will extend the docs. But then we should also change the docs on the existing methods using the input region like window and under_from_surface_tree?

Reply:
Sounds right, yeah

Reply:
I updated the docs on `Space::surface_under` and extended the docs on `Window::surface_under` and `under_from_surface_tree`.